templates/tfengine/recipes/devops.hcl

```diff
@@ -69,10 +76,17 @@ template "bootstrap" {
   output_path    = "./bootstrap"
 }
 
+{{if get . "enable_terragrunt" true}}
 template "root" {
   component_path = "../components/terragrunt/root"
   output_path    = "./live"
 }
+{{else}}
+template "root" {
+  component_path = "../components/terraform/root"
```

Comment:
Do we need a terraform root? I think in this case we should only generate the bootstrap and cicd dirs and leave the rest.

Reply:
yea, we actually still need it. The cloud build config need that to correctly determines where to find the terraform configs and its own config.

Reply:
oh sorry... I misunderstood. You mean the terraform root component here. I think having some terraform only root files can still be helpful. I added this terraform root to include a README file for terraform only useage and a main.tf which sets the backend and people can start adding their configs in that file.

Reply:
Does the solutions doc talk about how to manage multple deployments? Or are you supposed to put everything under the single live deployment?

Another caveat is all of our other templates will still continue to generate terragrunt.hcl files which will stop working, so we may need to think about this change a bit more. Worth a short meeting to discuss?

Reply:
Sure let’s chat tomorrow.

Reply:
Resolved offline.